packages/react-reconciler/src/ReactFiberBeginWork.js

```diff
@@ -885,6 +886,15 @@ export default function<T, P, I, TI, HI, PI, C, CC, CX, PL>(
     const newValue = newProps.value;
     workInProgress.memoizedProps = newProps;
 
+    if (__DEV__) {
+      checkPropTypes(
+        workInProgress.type.propTypes,
+        newProps,
+        'prop',
+        'Context.Provider',
+      );
```

Comment:
This is missing the component stack argument. Can you please add it, similar to how we do elsewhere?

I think you can use `getCurrentFiberStackAddendum` similar to how we do in `ReactDOMFiberComponent.js`.